practice/caesar/site.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+/**Returns the ASCII position of the inputted character.
+*/
+function getIndex(letter) {
+  return letter.charCodeAt();
+}
+
+/** Returns the ASCII character as a string associated with supplied index.
+*/
+function getLetter(index) {
+  return String.fromCharCode(index);
+}
+
+/** Takes in a string and shifts it [key] amount on the alphabet to encrypt
+the word.
+*/
+function caesarEncrypt(key, inputWord) {
+  var wordAsArray = inputWord.split('');
+  var indexArray = _.map(wordAsArray, getIndex);
+  var shiftedIndexArray = _.map(indexArray, function addKey(index) {
```

Comment:
Function definitions should either be function statements with a name, but not used immediately

```js
function addKey(index) {
...
}
```

or function expressions that don't have a name, but are used as a value immediately

```js
_.map(indexArray, function(index) {...});
```

I don't think it's actually syntactically correct JS, but the linter doesn't catch it.